src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -956,6 +964,10 @@ public Response assignGameBoard(@Context final HttpServletRequest request,
                 return new SegueErrorResponse(Status.FORBIDDEN, "You need a teacher account to create groups and set assignments!").toResponse();
             }
 
+            if (!isUserStaff(userManager, currentlyLoggedInUser) && (null == assignmentDTOFromClient.getNotes() || !assignmentDTOFromClient.getNotes().isEmpty())) {
```

Comment:
This blocks out anyone who isn't staff from creating an assignment! Conditionals need altering

Reply:
By design.

Reply:
This shouldn't be note dependant though, in this form it's stopping teachers from setting assignments to their groups of students, as they're not allowed to add notes, even though they don't/can't provide notes